README.md

```diff
@@ -77,7 +77,7 @@ This works on any deployment of Apache OpenWhisk
 
 To use on a deployment of OpenWhisk that contains the runtime as a kind:
 ```
-wsk action update helloJava hello.jar --main Hello --kind java:8
+wsk action update helloJava hello.jar --main Hello --kind java
```

Comment:
This is correct, the default runtime manifest does not specify `:8`. https://github.com/apache/incubator-openwhisk/blob/master/ansible/files/runtimes.json#L101

Reply:
heh we should change it for consistency.

Reply:
Ha, just posted that. @davidpodhola, probably better to change it here instead: https://github.com/apache/incubator-openwhisk/blob/master/ansible/files/runtimes.json#L101.

Reply:
OK, created https://github.com/apache/incubator-openwhisk/pull/4390